cmd/bnsd/app/app_performance_test.go

```diff
@@ -3,132 +3,164 @@ package app
 import (
 	"encoding/hex"
 	"io/ioutil"
-	"sync"
+	"os"
 	"testing"
 	"time"
 
 	"github.com/iov-one/weave"
 	"github.com/iov-one/weave/coin"
-	"github.com/iov-one/weave/errors"
 	"github.com/iov-one/weave/weavetest"
 	"github.com/iov-one/weave/x/cash"
 	"github.com/iov-one/weave/x/sigs"
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/log"
 )
 
-func BenchmarkBNSD(b *testing.B) {
+func BenchmarkBnsdEmptyBlock(b *testing.B) {
+	var aliceAddr = weavetest.NewKey().PublicKey().Address()
+
+	type dict map[string]interface{}
+	genesis := dict{
+		"gconf": map[string]interface{}{
+			cash.GconfCollectorAddress: hex.EncodeToString(aliceAddr),
+			cash.GconfMinimalFee:       coin.Coin{}, // no fee
+		},
+	}
+
+	bnsd, _ := newBnsd(b)
```

Comment:
I know... I wanted to figure out how to avoid that adding overhead to the benchmark.

Is there a parallel to b.ResetTimer() that ignores all time *after* it is called?